lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -359,9 +368,7 @@ def parse_kp(kp_filename, kp_specimen_manifest_filename, output):
     # Convert dtypes
     clinical_records["encountered"] = pd.to_datetime(clinical_records["encountered"])
 
-    # Insert static value columns
-    clinical_records["site"] = "KP"
-
+    clinical_records = insert_static_value_columns(clinical_records, {"site": "KP"})
```

Comment:
Ditto with this refactoring. It's not clear to me why this change is an improvement.

Reply:
Yeah.. this made more sense when we were adding multiple static values to a DataFrame across multiple lines, but the motivation is no longer clear to me for this refactor. I'm ok with dropping this commit.